### ISSUE TITLE
Analytics tab internal error

### ISSUE THREAD

**Issue by AesSedai** [wundergraph/cosmo#618]
### Component(s)

studio

### Component version

0.70.4

### wgc version

0.44.2

### controlplane version

0.74.1

### router version

0.72.0

### What happened?

If possible, please create a PR with a failing test to illustrate the issue clearly.
Otherwise, please attach a minimum reproduction through a GitHub repository that includes
essential information such as the relevant subgraph SDLs.
Please also make sure that the instructions for the reproduction are clear, tested, and fully accurate.

## Description
The Studio Analytics tab and traces tab show an internal server error when loading their pages

## Steps to Reproduce
Run the "Local Development" steps in the readme
Make a request in the Studio -> Playground tab
Navigate to the Studio -> Analytics tab, select Traces
=> Could not retrieve analytics data, [internal] internal error

## Expected Result
Studio -> Analytics tab should show Traces

## Actual Result
Studio -> Analytics tab shows an error

![image](https://github.com/wundergraph/cosmo/assets/7980540/ca7207d8-9d32-4f68-88d5-498cdcaf36a3)

## Environment
OS: WSL2 Ubuntu 20.04.5 LTS
Package Manager: pnpm: 8.15.4, nodejs: v20.11.1
Compiler(if manually compiled): go version go1.22.1 linux/amd64

### Router configuration

```yaml
Default "Local Development" setup
```

### Router execution config

```json
Default "Local Development" setup
```

### Log output

```shell
[23:28:53 UTC] INFO: incoming request {"reqId":"req-cq","req":{"method":"GET","url":"/wg.cosmo.platform.v1.PlatformService/GetAnalyticsView?connect=v1&encoding=json&message=%7B%22federatedGraphName%22%3A%22mygraph%22%2C%22config%22%3A%7B%22range%22%3A24%2C%22filters%22%3A%5B%7B%22field%22%3A%22operationName%22%2C%22value%22%3A%22MyQuery%22%7D%2C%7B%22field%22%3A%22operationHash%22%2C%22value%22%3A%2215855461482066297239%22%7D%5D%2C%22pagination%22%3A%7B%22limit%22%3A20%7D%2C%22sort%22%3A%7B%22id%22%3A%22unixTimestamp%22%2C%22desc%22%3Atrue%7D%7D%2C%22namespace%22%3A%22default%22%7D","hostname":"localhost:3001","remoteAddress":"127.0.0.1","remotePort":41634}}
[23:28:53 UTC] ERROR: Code: 352. DB::Exception: Block structure mismatch in (columns with identical name must have identical structure) stream: different types:
OrganizationID String String(size = 0)
OrganizationID LowCardinality(String) ColumnLowCardinality(size = 0, UInt8(size = 0), ColumnUnique(size = 1, String(size = 1))). (AMBIGUOUS_COLUMN_NAME) (version 24.2.1.2248 (official build))

[23:28:53 UTC] ERROR: {"reqId":"req-cq","service":"wg.cosmo.platform.v1.PlatformService","method":"GetAnalyticsView","actor":{"userId":"a75303d7-df64-4c64-b25b-847431a1621d","organizationId":"9be50f06-abc7-43ed-a2e2-01ede212677d"}}
[23:28:53 UTC] INFO: request completed {"reqId":"req-cq","res":{"statusCode":500},"responseTime":13.653982043266296}
[23:28:53 UTC] ERROR: Code: 352. DB::Exception: Block structure mismatch in (columns with identical name must have identical structure) stream: different types:
OrganizationID String String(size = 0)
OrganizationID LowCardinality(String) ColumnLowCardinality(size = 0, UInt8(size = 0), ColumnUnique(size = 1, String(size = 1))). (AMBIGUOUS_COLUMN_NAME) (version 24.2.1.2248 (official build))
```

**Comment by AesSedai** [comment]
This also shows up as missing info for the Top 10 Operations and Requests sections in the graph overview page:
![image](https://github.com/wundergraph/cosmo/assets/7980540/fa451b15-9cd6-4727-a1a1-721fc9d1a4c5)

```
[23:35:08 UTC] INFO: incoming request {"reqId":"req-en","req":{"method":"GET","url":"/wg.cosmo.platform.v1.PlatformService/GetDashboardAnalyticsView?connect=v1&encoding=json&message=%7B%22federatedGraphName%22%3A%22mygraph%22%2C%22range%22%3A4%2C%22namespace%22%3A%22default%22%7D","hostname":"localhost:3001","remoteAddress":"127.0.0.1","remotePort":45800}}
[23:35:08 UTC] ERROR: Code: 352. DB::Exception: Block structure mismatch in (columns with identical name must have identical structure) stream: different types:
SubgraphID LowCardinality(String) ColumnLowCardinality(size = 0, UInt8(size = 0), ColumnUnique(size = 1, String(size = 1)))
SubgraphID String String(size = 0). (AMBIGUOUS_COLUMN_NAME) (version 24.2.1.2248 (official build))

[23:35:08 UTC] ERROR: {"reqId":"req-en","service":"wg.cosmo.platform.v1.PlatformService","method":"GetDashboardAnalyticsView","actor":{"userId":"a75303d7-df64-4c64-b25b-847431a1621d","organizationId":"9be50f06-abc7-43ed-a2e2-01ede212677d"}}
[23:35:08 UTC] INFO: request completed {"reqId":"req-en","res":{"statusCode":500},"responseTime":19.40735900402069}
[23:35:08 UTC] ERROR: Code: 352. DB::Exception: Block structure mismatch in (columns with identical name must have identical structure) stream: different types:
SubgraphID LowCardinality(String) ColumnLowCardinality(size = 0, UInt8(size = 0), ColumnUnique(size = 1, String(size = 1)))
SubgraphID String String(size = 0). (AMBIGUOUS_COLUMN_NAME) (version 24.2.1.2248 (official build))
```

**Comment by StarpTech** [comment]
Hi @AesSedai, thanks for reporting. We will look into it.

**Comment by thisisnithin** [comment]
Hi @AesSedai, seems we are having this issue from ClickHouse version 24.2 onwards. While we work on a fix, please use version <= 24.1.

**Comment by thisisnithin** [comment]
This is fixed in the latest version of clickhouse. Please reopen if this occurs again.
